data/scenarios/Challenges/Sliding Puzzles/3x3.yaml

```diff
@@ -0,0 +1,742 @@
+      - [17, q-tile-ordinal]
+      - [18, r-tile-ordinal]
+      - [19, s-tile-ordinal]
+      - [20, t-tile-ordinal]
+      - [21, u-tile-ordinal]
+      - [22, v-tile-ordinal]
+      - [23, w-tile-ordinal]
+      - [24, x-tile-ordinal]
+      - [25, y-tile-ordinal]
+    program: |
+      run "scenarios/Challenges/Sliding Puzzles/_sliding-puzzle/maintainer.sw";
+  - name: setup
+    system: true
+    dir: [1, 0]
+    display:
+      invisible: true
+      attr: 'iron'
+    inventory:
+      - [1, a-tile-ordinal]
+      - [2, b-tile-ordinal]
+      - [3, c-tile-ordinal]
+      - [4, d-tile-ordinal]
+      - [5, e-tile-ordinal]
+      - [6, f-tile-ordinal]
+      - [7, g-tile-ordinal]
+      - [8, h-tile-ordinal]
+      - [9, i-tile-ordinal]
+      - [10, j-tile-ordinal]
+      - [11, k-tile-ordinal]
+      - [12, l-tile-ordinal]
+      - [13, m-tile-ordinal]
+      - [14, n-tile-ordinal]
+      - [15, o-tile-ordinal]
+      - [16, p-tile-ordinal]
+      - [17, q-tile-ordinal]
+      - [18, r-tile-ordinal]
+      - [19, s-tile-ordinal]
+      - [20, t-tile-ordinal]
+      - [21, u-tile-ordinal]
+      - [22, v-tile-ordinal]
+      - [23, w-tile-ordinal]
+      - [24, x-tile-ordinal]
+      - [25, y-tile-ordinal]
+    program: |
+      run "scenarios/Challenges/Sliding Puzzles/_sliding-puzzle/setup.sw";
+solution: |
+   run "scenarios/Challenges/Sliding Puzzles/_sliding-puzzle/solution.sw"
+entities:
+  - name: gradiator
+    display:
+      char: 'G'
+    description:
+      - Generates a magnetic field gradient that allows the use of `resonate`
+    properties: [known]
+    capabilities: [detectcount]
+  - name: locator
+    display:
+      char: '{'
+    description:
+      - Enables the `detect` command
+    properties: [known]
+    capabilities: [detectloc]
+  - name: border
+    display:
+      char: '▒'
+    description:
+      - Immovable playfield border
+    properties: [known]
+  - name: ink
+    display:
+      char: 'i'
+    description:
+      - Marking fluid to designate the tile to slide
+    properties: [known]
+  - name: dozer blade
+    display:
+      attr: silver
+      char: '/'
+    description:
+    - Facilitates pushing
+    properties: [known, portable]
+    capabilities: [push]
+  - name: sliding-tile
+    display:
+      char: '*'
+    description:
+      - Tile that is being moved
+    properties: [known]
+  - name: a-tile
+    display:
+      char: 'a'
+      attr: oddtile
+    description:
+      - One
+    properties: [known, portable]
+  - name: b-tile
+    display:
+      char: 'b'
+      attr: eventile
+    description:
+      - Two
+    properties: [known, portable]
+  - name: c-tile
+    display:
+      char: 'c'
+      attr: oddtile
+    description:
+      - Three
+    properties: [known, portable]
+  - name: d-tile
+    display:
+      char: 'd'
+      attr: eventile
+    description:
+      - Four
+    properties: [known, portable]
+  - name: e-tile
+    display:
+      char: 'e'
+      attr: eventile
```

Comment:
How come `d-tile` and `e-tile` are both `eventile`?

Reply:
Actually it's not just `d` and `e` that seem off, I don't understand the rationale for which tiles are "even" and "odd" at all.

Reply:
Good catch.  I originally wanted the solved 4x4 puzzle to have a checkerboard-like appearance, which meant that every other row would have to invert the parity-based color.  But then I realized that this would not hold for a 3x3 or 5x5 puzzle so I kind of threw up my hands.

Now the tiles are consistently colored based on parity.  A solved 3x3 puzzle or 5x5 puzzle will appear as a checkerboard, whereas solved 4x4 would have vertical stripes.